src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,8 +1,24 @@
-export const TodoFilter = () => (
+type Props = {
+  query: string;
+  setQuery: (value: string) => void;
+  selectedFilter: string;
+  setSelectedFilter: (value: string) => void;
+};
+
+export const TodoFilter: React.FC<Props> = ({
+  query,
+  setQuery,
+  selectedFilter,
+  setSelectedFilter,
```

Comment:
when you pass a setter to a function, it is better to wrap it in another function or at least rename it while passing to onName